omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -52,6 +52,15 @@
             delete "#{install_dir}/bin/agent/dist/*.conf*"
             delete "#{install_dir}/bin/agent/dist/*.yaml"
             command "del /q /s #{windows_safe_path(install_dir)}\\*.pyc"
+
+            # On Windows, zip up the python directory
+            command "7z a -mx=5 -ms=on #{install_dir}/embedded3.7z #{windows_safe_path(python_3_embedded)}"
```

Comment:
Can you add a comment explaining the use of `-mx=5 and -ms=on`? It's not obvious what they do or why we need them.